eng/AzSdk.reference.targets

```diff
@@ -3,7 +3,7 @@
     <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="[3.3.18, 4.0.0)" />
     <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="[2.3.18, 3.0.0)" />
   </ItemGroup>
-
+  
   <ItemGroup Condition="'$(IsTargetingNetStandard)' == 'true'">
```

Comment:
Looks like there is still some whitespaces changes in this file.

Reply:
it's exactly the same as in the master branch though.

Reply:
I see an extra space in the diff here on github.

Reply:
Oh, so sorry. I have fixed that now.